src/Components/PresentationalComponents/Filters/PrimaryToolbarFilters/OsVersionFilter.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { RHEL_VERSIONS } from '../../../../Helpers/constants';
+import { conditionalFilterType } from '@redhat-cloud-services/frontend-components/ConditionalFilter';
+import { Select, SelectOption } from '@patternfly/react-core';
+import { intl } from '../../../../Utilities/IntlProvider';
+import messages from '../../../../Messages';
+
+const useOsVersionFilter = (urlParam, label, placeholder, filterValue, apply) => {
+    const [isOpen, setOpen] = React.useState(false);
+    const [numOptions, setNumOptions] = React.useState(10);
+    const versionFromNewestToOldest = RHEL_VERSIONS.slice().reverse();
+
+    let selectedVersionsArray = filterValue ? filterValue.split(',') : [];
+
+    const onOptionSelect = (_, changedValue) => {
+        let newValue;
+
+        if (selectedVersionsArray.includes(changedValue)) {
+            newValue = selectedVersionsArray;
+            newValue.splice(newValue.indexOf(changedValue), 1);
+        }
+        else {
+            newValue = [...selectedVersionsArray, changedValue];
+        }
+
+        apply({ [urlParam]: newValue.join(','), page: 1 });
+    };
+
+    const onToggle = (isOpen) => {
+        setOpen(isOpen);
+    };
+
+    const onViewMoreClick = () => {
+        setNumOptions(versionFromNewestToOldest.length);
+    };
+
+    return {
+        type: conditionalFilterType.custom,
+        label: intl.formatMessage(label),
+        filterValues: {
+            children: (
+                <Select
+                    variant={'checkbox'}
```

Comment:
Let's try to avoid hardcoded variants and use `SelectVariant`

Reply:
I wanted to, but there seems to be an import bug in current PF version preventing me from using SelectVariant.checkbox